rest/server_context.go

```diff
@@ -87,18 +87,24 @@ func NewServerContext(config *ServerConfig) *ServerContext {
 	return sc
 }
 
+// PostStartup runs anything that relies on SG being fully started (i.e. sgreplicate)
 func (sc *ServerContext) PostStartup() {
+	// Introduce a minor delay if there are any replications
+	// (sc.startReplicators() might rely on SG being fully started)
+	if len(sc.config.Replications) > 0 {
```

Comment:
We're doing a length check here, and a nil check in startReplicators().  Looks like the nil check in startReplicators is incorrect - want to clean that up while you're here?

Reply:
Done, thanks.